rwengine/src/data/GameTexts.hpp

```diff
@@ -28,6 +28,19 @@ namespace GameStringUtil
 GameString fromString(const std::string& str);
 }
 
+/**
+ * Since the encoding of symbols is arbitrary, these constants should be used in
+ * hard-coded strings containing symbols outside of the ASCII-subset supported by
+ * all fonts
+ */
+namespace GameSymbols
+{
+	static constexpr GameStringChar Money  = '[';
+	static constexpr GameStringChar Heart  = '{';
+	static constexpr GameStringChar Armour = '{';
+	static constexpr GameStringChar Star   = ']';
+}
```

Comment:
nit: imo these could just be part of the GameString class in a Symbol namespace (Say GameString::Symbol::Heart) OR if we typedef to a GameStringChar they could be part of that class too (GameStringChar::Heart)?
Adding too many namespaces / classes for something this simple might be confusing.